project1/styles.css

```
@@ -0,0 +1,86 @@
+blockquote p {
```

I know you also spotted that the horizontal margins of this are out of whack with the rest of your document. The `blockquote` element has margins by default, so you can fix this by overriding those margins:
```suggestion
blockquote {
    margin: 0;
}
blockquote p {
```

---

I'm glad I changed this, I kept having issues because I was not paying attention to the "blockquote p" element and I was only seeing it as a "blockquote" element.